containers/datalab/content/run.sh

```diff
@@ -54,6 +54,21 @@ check_tmp_directory() {
     echo "The /tmp directory is writable"
 }
 
+# Reinstall the parts of pydatalab from /content/pydatalab, which is where it gets live-mounted.
+function reinstall_pydatalab() {
+  PYDATALAB=/content/pydatalab
+  echo "Reinstalling pydatalab from ${PYDATALAB}"
+  pip uninstall -y datalab
```

Comment:
I don't think you need the uninstall, `--force-reinstall` or `--upgrade` will overwrite.